### ISSUE TITLE
Request: Elementor Icon

### ISSUE THREAD

**Issue by AimarBustamante** [simple-icons/simple-icons#11122]
### Brand Name

Elementor

### Website

https://elementor.com/

### Popularity Metric

https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/3m?webSource=Total&key=elementor.com 

### Official Resources for Icon and Color

~~SVG: https://uxwing.com/elementor-icon/~~
See below for official sources.

**Comment by adamrusted** [comment]
There are a collection of official SVG logos available [here](https://elementor.com/logos), and guidelines available at the same link.

**Comment by anhthang** [comment]
@adamrusted this logo already available in the website

![image](https://github.com/simple-icons/simple-icons/assets/10458061/9c0a92b0-3d9a-4c95-ada2-53e71c05bf90)

**Comment by adamrusted** [comment]
And there's me just checking our historic issues - thanks for the assist @anhthang!

**Comment by AimarBustamante** [comment]
Hi, sorry but I don't see the badge on the repo

**Comment by adamrusted** [comment]
https://github.com/simple-icons/simple-icons/blob/develop/icons/elementor.svg
https://simpleicons.org/?q=elementor

**Comment by AimarBustamante** [comment]
I was referring to this 
![image](https://github.com/simple-icons/simple-icons/assets/106636449/e972f9de-ab97-4425-8c08-4d228ff7b3a9)

**Comment by adamrusted** [comment]
Ah, we don't handle shields/badges - they're managed over at https://github.com/badges/shields.
Which repo are you looking at for specific examples like that? Best to reach out to that repo owner and ask them to create an Elementor one, or create the URL yourself based on the icon slug.

**Comment by adamrusted** [comment]
Ah, I see! Best creating an issue on that specific repo then - as they'll be able to add it to the list. _OR_ you can use the UI available [here](https://shields.io/badges/static-badge) to generate it yourself, using the info found [here](https://simpleicons.org/?q=elementor).

**TL;DR**
We're responsible for the collection of brand icons.
Shields.io is the image service that repo is using.
The repo owner can generate an Elementor badge to include in that repo, should they choose.

**Comment by AimarBustamante** [comment]
Oh, I see. Ty for the info. Will do that then.
